sycl/include/CL/sycl/types.hpp

```diff
@@ -357,7 +357,7 @@ template <typename Type, int NumElements> class vec {
   using vector_t = DataType;
 #endif
 
-  vec() { m_Data = {0}; }
+  vec() = default;
```

Comment:
Can you cite a spec to confirm that "This initialization is unnecessary"?

Reply:
From Table 4.95: Constructors of the SYCL vec class template.

> vec() - Default construct a vector with element typedataT and with numElements dimensions bydefault construction of each of its elements.